src/aks-preview/azext_aks_preview/custom.py

```diff
@@ -483,36 +483,40 @@ def aks_update(cmd, client, resource_group_name, name, enable_cluster_autoscaler
                disable_cluster_autoscaler=False,
                update_cluster_autoscaler=False,
                min_count=None, max_count=None, no_wait=False):
-    if (all([enable_cluster_autoscaler, disable_cluster_autoscaler]) or
-            all([enable_cluster_autoscaler, update_cluster_autoscaler]) or
-            all([disable_cluster_autoscaler, update_cluster_autoscaler])):
-        raise CLIError('specify either "--enable_cluster_autoscaler" or '
-                       '"--disable_cluster_autoscaler" or '
-                       '"--update_cluster_autoscaler".')
-    if all([not enable_cluster_autoscaler, not disable_cluster_autoscaler, not update_cluster_autoscaler]):
-        raise CLIError('specify either "--enable_cluster_autoscaler" or '
-                       '"--disable_cluster_autoscaler" or '
-                       '"--update_cluster_autoscaler".')
+    update_flags = []
```

Comment:
It actually just ocurred to me that if you just add all of them up it will work
```flags = enable_cluster_autoscaler + disable_cluster_autoscaler + update_cluster_autoscaler```

Boolean true will convert to 1 and false to 0. Thanks to python duck typing I guess

Reply:
Sorry, for iterating so much on this.

Reply:
Great, I didn't know this. Very glad to learn new. :)